### ISSUE TITLE
Gadfly does not graph and run time issue.

### ISSUE THREAD

**Issue by ccsv** [quinnj/Sublime-IJulia#62]
So I have an issue executing Julia code on sublime takes around more than 20 seconds. Compare the same code in python in sublime text takes 3 seconds.

Second problem is Gadfly graphs do not display at all

**Comment by quinnj** [comment]
Yep, no graphing works in sublime for the moment (see #37). As far as the speed of the Julia code, you'll have to post the code you're running if you'd like me to help debug, otherwise I have no idea what's going on.
